CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 - Add missing fields to GPUContext ([#1391](https://github.com/getsentry/relay/pull/1391))
 - Store actor now uses Tokio for message handling instead of Actix. ([#1397](https://github.com/getsentry/relay/pull/1397))
 - Add app_memory to AppContext struct. ([#1403](https://github.com/getsentry/relay/pull/1403))
+- Project::state() return None for expired projects. ([#1404](https://github.com/getsentry/relay/pull/1404))
```

Comment:
super _nit_ 😄: Could we write this even more from a user's perspective and add this to bugfixes? The changed behavior is now that we no longer apply outdated rate limits or reject data after the cache has expired.

Reply:
Done.